src/components/input/MultiSelect.vue

```diff
@@ -170,6 +178,7 @@ const pt = computed(() => ({
       selectedCount.value > 0
         ? 'border-blue-400 dark-theme:border-blue-500'
         : 'border-transparent',
+      'focus-within:border-blue-400 dark-theme:focus-within:border-blue-500',
```

Comment:
The border color is being set in two places. It makes sense that focus-within would take precedence.
Want to wrap this in a `cn()` to make sure the options merge in order?

Reply:
Actually, it's a little weird that we'd have focus states on both the trigger and the menu item...